packages/storefront/src/pages/styles/skeleton.md

```diff
@@ -0,0 +1,53 @@
+# Skeleton
+
+<TableOfContents></TableOfContents>
+
+## Example
+
+<Playground :frameworkMarkup="codeExample" :externalStackBlitzDependencies="['styled-components']">
+  <ExampleStylesSkeleton />
+</Playground>
+
+## Usage
+
+The skeleton styles can be used to illustrate the loading state of more complex elements, for example a list of cards.
+
```

Comment:
Do we also need "Do/Don't" section?

Reply:
I don't think we got this for any other style.